ccp/modules/mtba-setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ "$ENABLE_MTBA" == true ]; then
```

Comment:
Is there some other variable (other than this explicit one) that could serve to detect MTBA mode?

Reply:
Unfortunately, it is not possible. There are some environment variables that can be set, but they are all optional.